packages/TokenRunner/src/Wrapper/FixerWrapper/PropertyWrapper.php

```diff
@@ -64,8 +57,9 @@ public function getDocBlockWrapper(): ?DocBlockWrapper
 
     protected function getNamePosition(): int
     {
-        $nextVariableTokens = $this->tokens->findGivenKind([T_VARIABLE], $this->index, $this->index + 5);
+        $expressionEndPosition = $this->tokens->getNextTokenOfKind($this->index, [';']);
```

Comment:
This `+ 5` bulgarian constant caused token overflow. E.g. token 40 → 45, but file ended at 44.